### PR TITLE
Tests impl

### PR DESCRIPTION
**Pull request by EloiGG**

### CODE REVIEW

**Comment by EloiGG** [comment]
keep my gitignore, it ignores all visual studio files, including my projects

**Review by LazyKeru — COMMENTED** [review]
".vs/SmellsFishy/v17/.suo" and ".vs/slnx.sqlite" are not necessary ? Could be deleted in the near future

File: CppProperties.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (LazyKeru) [inline CppProperties.json:1]: Do we need this file ? As not everyone in the project uses Visual Studio

File: src/test.cpp [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+			test::regex_test();
+		}
+		else if (s == "entropy") {
+			
+		}
+		else if (s == "file_read") {
+			test::io_test();
+		}
+	}
+}
```
Comment (LazyKeru) [inline src/test.cpp:26]: Nicely done, we can then add it into a workflow

File: src/io/io.cpp [hunk 3]
```diff
@@ -1,7 +1,7 @@
-#include "io.h"
+#include "../../header/io/io.h"
 #include <sstream>
 #include <fstream>
-#include "util.h"
+#include "../../header/util/util.h"
 
 std::string io::fileToString(const std::string& filePath)
 {
```
Comment (LazyKeru) [inline src/io/io.cpp:7]: This file was renamed in a previous change

**Review by LazyKeru — CHANGES_REQUESTED** [review]
Need to be up to date with the rest of the projecct

**Comment by LazyKeru** [comment]
Fixed merge conflict and cleaned the directory. The new working branch: [feat-test-function](https://github.com/LazyKeru/SmellsFishy/tree/feat-test-function)

**Pull request closed without merging**